spec/shoryuken/polling_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require 'shoryuken/polling'
+
+describe Shoryuken::Polling do
+  let(:queue1) { "shoryuken" }
+  let(:queue2) { "uppercut" }
+  let(:queues) { Array.new }
+
+  describe Shoryuken::Polling::WeightedRoundRobin do
+    subject { Shoryuken::Polling::WeightedRoundRobin.new(queues) }
+
+    it "decreases weight" do
+      # [shoryuken, 2]
+      # [uppercut,  1]
+      queues << queue1
+      queues << queue1
+      queues << queue2
+
+      expect(subject).to eq [queue1, queue2]
+
+      subject.pause(queue1)
+
+      expect(subject).to eq [queue2]
+    end
+
+    it "increases weight" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.